transport/tchannel/peer.go

```diff
@@ -113,6 +114,14 @@ func (p *tchannelPeer) release() {
 }
 
 func (p *tchannelPeer) setConnectionStatus(status peer.ConnectionStatus) {
+	p.transport.logger.Info(
+		"connection status change",
+		zap.String("status", status.String()),
+		zap.String("address", p.Peer.Identifier()),
+		zap.String("localAddress", p.transport.addr),
```

Comment:
Isn't this the transport's listening address and irrelevant to the connecting peer? 
```suggestion
```